src/routes/docs/tutorials/sveltekit-ssr-auth/step-1/+page.markdoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: tutorial
+title: Server Side Rendering with SvelteKit
+description: Add SSR Authentication to your SvelteKit app with Appwrite
+step: 1
+difficulty: beginner
+readtime: 20
+framework: SvelteKit
+category: Auth
+---
+
+Appwrite takes away the stress of building and maintaining a backend. Appwrite helps implement authentication, databases, file storage, and respond to real-time events with **secure** APIs out of the box. 
+If you're a Svelte developer, the examples in this guide show you how Appwrite can help you add authentication to Svelte apps faster.
+
+## Before you start
```

Comment:
Please use `#` instead of `##` because internal logic demotes tags by one level.

Also check for tags where you don't have {% #link-to-header %}

Reply:
Done